geode-docs/developing/query_select/the_where_clause.html.md.erb

```diff
@@ -235,6 +235,35 @@ If more than one method can be invoked, the query processor chooses the method w
 
 The query processor uses the runtime types of the parameters and the receiver to determine the proper method to invoke. Because runtime types are used, an argument with a null value has no typing information, and so can be matched with any object type parameter. When a null argument is used, if the query processor cannot determine the proper method to invoke based on the non-null arguments, it throws an `AmbiguousNameException`.
 
+**Methods calls with the `SecurityManager` enabled**
+
+When the `SecurityManager` is enabled,
+<%=vars.product_name%> throws a `NotAuthorizedException` when any
+method not on this list of whitelisted methods is invoked:
+
+- `keySet`, `entrySet`, `values`, `containsKey` or `get` on a `Map`, `Collection`, or `Region` object
+- `getKey` or `getValue`  on a `Region.Entry` object
+- `after`, `before`, `getNanos`, or `getTime` on a `Date` or `Timestamp` object
+- `charAt`, `codePointAt`, `codePointBefore`, `codePointCount`, 
+`compareToIgnoreCase`, `concat`, `contains`, `contentEquals`, 
+`endsWith`, `equalsIgnoreCase`, `getBytes`, `hashCode`, `indexOf`, 
+`intern`, `isEmpty`, `lastIndexOf`, `length`, `matches`, 
+`offsetByCodePoints`, `regionMatches`, `replace`, `replaceAll`, 
+`replaceFirst`, `split`, `startsWith`, `substring`, `toCharArray`, 
+`toLowerCase`, `toUpperCase`, or `trim` on a `String` object
+- `intValue`, `longValue`, `shortValue`, etc. on any `Number` object
+- `booleanValue` on any `Boolean` object
+- `equals`, `compareTo`, or `toString` on any object
+
+To disable the authorization check,
+start all servers with the added system property
+`QueryService.allowUntrustedMethodInvocation`.
+For example,
+
+```
+gfsh>start server --name=Server1 --J=QueryService.allowUntrustedMethodInvocation
```

Comment:
I think this should be -J=-DQueryService.allowUntrustedMethodInvocation

Reply:
Two hyphens before the "J" is correct in a gfsh command. I think Dan just meant to add the "-D".

Reply:
Good catch.  Thanks.  In fact, I think that the option should read
`--J=-Dgemfire.QueryService.allowUntrustedMethodInvocation=true`